dev-util/idea-ultimate/idea-ultimate-15.0.0.142.3926.4_pre.ebuild

```diff
@@ -11,11 +11,11 @@ MY_PN="idea"
 
 DESCRIPTION="A complete toolset for web, mobile and enterprise development"
 HOMEPAGE="http://www.jetbrains.com/idea"
-SRC_URI="https://download.jetbrain.com/idea/${MY_PN}IU-${MY_PV}.tar.gz"
+SRC_URI="https://download.jetbrains.com/idea/${MY_PN}IU-${MY_PV}.tar.gz"
 
 LICENSE="IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal"
 IUSE=""
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="-amd64 -x86"
```

Comment:
Am I allowed to set the stable version to stable in keywords? didn't do that because I am not part of the group of gentoo-developpers, but -amd64 is indeed a bad solution because it does work.